app/src/main/java/com/duckduckgo/app/browser/TabSwitcherButton.kt

```diff
@@ -35,46 +43,33 @@ class TabSwitcherButton(context: Context) : FrameLayout(context) {
     var hasUnread = false
         set(value) {
             field = value
-            anim.progress = if (hasUnread) 1.0f else 0.0f
         }
 
-    init {
-        LayoutInflater.from(context).inflate(R.layout.view_tab_switcher_button, this, true)
+    override fun onFinishInflate() {
+        super.onFinishInflate()
+        View.inflate(context, R.layout.view_tab_switcher_button, this)
 
-        clickZone.setOnClickListener {
+        setOnClickListener {
             super.callOnClick()
         }
 
-        clickZone.setOnLongClickListener {
+        setOnLongClickListener {
             super.performLongClick()
         }
     }
 
     fun increment(callback: () -> Unit) {
-        anim.progress = 0.0f
```

Comment:
as per new designs, we've removed the animation here so we can have proper light / dark theme support.